src/adapter.js

```diff
@@ -52,6 +52,7 @@ function createQUnitStartFn (tc, runnerPassedIn) { // eslint-disable-line no-unu
             fixture.style.width = '1000px'
             fixture.style.height = '1000px'
             document.body.appendChild(fixture)
+            runner.config.fixture = fixture
```

Comment:
This is documented as taking a string or `null`. Storing a Node here is not a public interface currently. It appears to be an internal detail that was introduced in a recent release, which may not be compatible with other 2.x versions of QUnit.

Reply:
It has been introduced in `qunit@2.5.1`  - I can create an issue in QUnit to see if they will document it as a public interface. In `2.5.0` they've changed the name was from `qunitjs` to `qunit` which means we don't really need to worry about backwards compatibility.